Source/Applications/SystemCenter/wwwroot/Scripts/TSX/SystemCenter/NewMeterWizard/NewMeterWizard.tsx

```diff
@@ -94,10 +94,12 @@ export default function NewMeterWizard(props: {IsEngineer: boolean}) {
     // Wizard Page Control
     const [error, setError] = React.useState<string[]>([]);
     const [warning, setWarning] = React.useState<string[]>([]);
-    const [hover, setHover] = React.useState<'None' | 'Next' | 'Prev'>('None');
+    const [hover, setHover] = React.useState<'None' | 'Next' | 'Prev' | 'Drawings'>('None');
     const [showSubmit, setShowSubmit] = React.useState<boolean>(false);
     const [status, setStatus] = React.useState<Application.Types.Status>('unintiated');
 
+    const locationInfoArray = React.useMemo(() => [locationInfo], [locationInfo]);
```

Comment:
seems weird you keep memonizing these states. Is there something causing this to be neccesarry?

Reply:
It was causing excessive re renders every time the parent was interacted with